### ISSUE TITLE
Incorrect result

### ISSUE THREAD

**Issue by jly0810** [Dongjiahua/VICA-NeRF#5]
I tested according to your method (performed 2x downsampling), but the result shown below is not good.
1、"Give him a cowboy hat"
![frame_00001](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/89c61318-7f63-417b-96e8-0c6d2bab344b)
![frame_00008](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/1079fca3-8924-4aa4-91c9-66c5085a96a2)
![frame_00033](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/dee3fb9c-97bd-4405-b60b-71eebfc3912b)
2、“Turn him into Spider man with mask”
![frame_00001](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/7bace975-0b57-4744-b539-d8a93c26d5f4)
![frame_00019](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/7b49d672-4260-4229-85e0-ee2d923417ef)
May I ask what the reason is? I have not made any modifications to your code and parameters. I conducted the experiment using version 1.1.0 of nerfstudio on Windows. I hope you can answer any doubts and provide a generation plan that is consistent with your paper's results. Thank you.

**Comment by Dongjiahua** [comment]
Thanks for your question; we noticed changes in the latest Nerfstudio version. Please use nerfstudio==1.0.0 when installing it. This could correct the result. Our experiments are based on Linux, so Windows' performance is not guaranteed, but it should look similar.

To help you achieve similar results in the paper, we updated the code to use a certain random seed (please redownload the code and install it). The checkpoint of the original scene can be downloaded [here](https://drive.google.com/file/d/18-z98XR4QHK2TD97JNFtbK5ga5RCsRnb/view?usp=sharing). I just rerun these two experiments to see the results.

For spider-man, use `seed = 5` and `post_refine=True`, the results should like:
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/48640121/871cfcfb-15b2-4963-b46c-895430672c07)


For wearing the hat, use `seed = 5` and `post_refine=True`, the results should like:
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/48640121/6fd4cc1a-d47e-4644-9545-3cc46eae90fe)


Note that these are not the optimal results. You can also try other parameters for better performance. To achieve the best performance, you can use the "control" mode to avoid incorrect editing from ip2p.

**Comment by jly0810** [comment]
Thank you for your reply. I used the checkpoints you provided for the first stage and obtained results similar to yours. However, using the results of the first stage of my own training resulted in a serious lack of facial details in the final editing result. Why is this? Can you provide training commands for your checkpoints? This is the training command I used: ns-train nerfacto --data ./data/face/face nerfstudio-data --downscale-factor 2
The checkpoint of the original scene can be downloaded：https://drive.google.com/drive/folders/1w5MZ7CHYr7zrRcFCPEFJleMqrx96akTE?usp=sharing
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/04718866-f7f5-4987-ab86-b09017ff5565)

**Comment by Dongjiahua** [comment]
Hi, with the updated Nerfstudio of version 1.0.0, you may also need to **retrain** the first stage with it. Otherwise, the loading will be incorrect.

**Comment by jly0810** [comment]
> 您好，Nerfstudio 已经更新至 1.0.0 版本，您可能还需要用它**重新训练**第一阶段。否则，加载将不正确。

Yes, the above result is obtained using nerfstudio=1.0.0. There are some issues with the final editing effect.

**Comment by Dongjiahua** [comment]
Could you try other prompts like "turn him into Iron Man" and show the results?

**Comment by jly0810** [comment]
"turn him into Iron Man" 
use seed = 5 and post_refine=True
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/997aa57c-0828-4770-a62e-242581714b50)

**Comment by jly0810** [comment]
"turn him into Iron Man"
Use the first stage model you provided：
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/052dc282-6494-4266-97b9-a091918086c8)


“Give him a mustache”
Use the first stage model you provided：
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/104cd246-cf3a-4361-acf5-b132fec1cfbd)
Use the first stage model generated by my training：
![image](https://github.com/Dongjiahua/VICA-NeRF/assets/89561918/e152383d-96cb-47d0-9be5-b2abf825143a)

**Comment by Dongjiahua** [comment]
I don't see a problem with your training code, but your pretrained checkpoint seems to have incorrect depth estimations, as the inconsistent result showed:
![0_30000](https://github.com/Dongjiahua/VICA-NeRF/assets/48640121/0d1169c0-6e7c-4183-b6b3-3a3d64ca4253)
![9_30000](https://github.com/Dongjiahua/VICA-NeRF/assets/48640121/50d0ca85-92d7-4851-9b29-edf963c0adc9)



There could be several reasons
1. You don't need to downsample the image during pretraining, and avoid doing this can somehow improve the performance
2. Other parameters: there are other parameters that need to be set during pretraining, depending on specific scenes (e.g., near surface). Please refer to [Instruct-NeRF2NeRF](https://instruct-nerf2nerf.github.io/)'s settings.

For our checkpoint, you can get similar results to the paper for "Iron Man" by using "seed=0". The results are like:
![render_14](https://github.com/Dongjiahua/VICA-NeRF/assets/48640121/6ea7bbd7-9f62-4373-aa32-8255fc167ff9)
![render_43](https://github.com/Dongjiahua/VICA-NeRF/assets/48640121/d09bf613-840f-4c71-bb30-b5005a066f55)

(Note that the results are suboptimal since I just run 3000 steps for a quick view)

**Comment by jly0810** [comment]
Can you provide the training parameters for your pre-trained model？
IN2N only provided other training hyperparameter settings during the editing phase.

**Comment by Dongjiahua** [comment]
For this checkpoint, I just simply use 

ns-train nerfacto --data ./data/face/face

IN2N describe some pretrain setting in their issues. Since these data come from them and they are also the authors of nerfstudio, they can have better idea of the settings.

**Comment by jly0810** [comment]
I think I know the problem now. I previously downloaded the development version of nerfstudio (modified version number to 1.0.0 in pyproject.toml). I just tried using pip install nerfstudio=1.0.0 to download, and the result is correct. Do you know how to download nerfstudio1.0.0 through pip install - e. (because the default is to download the latest version)

**Comment by Dongjiahua** [comment]
Sounds good. You can download the source from their 1.0.0 release on github
